conformance/ion_encoding/module/macro/template/if.ion

```diff
@@ -0,0 +1,116 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+
+(ion_1_1 "if_void"
+
+  (then "happy path"
+    (mactab (macro m (v*)
+              (if_void v true false)))
+    (toplevel ('#$:m') ('#$:m' 1) ('#$:m' 1 2) ('#$:m' 1 2 3))
+    (produces true     false      false        false))
+
+  (then "then-clause isn't evaluated unless selected"
+    (mactab (macro m (p*)
+              (if_void p (make_string null) false)))
```

Comment:
Let's put something a little more obvious here and save `(make_string null)` for [this file](https://github.com/amazon-ion/ion-tests/blob/master/conformance/eexp/basic_system_macros.ion).
```suggestion
              (if_void p (make_string {}) false)))
```